### PR TITLE
Issue #35 - Consolidate JS

### PR DESCRIPTION
**Pull request by sesheppard**
Fixing Issue #35 - Working on separating in-line javascript functions from each HTML document into separate files. The footer and sidebar functionality appear on every page, so they will be stored in the playground.js file. The Tools page contained many of its own functions, so I am moving them to the tools.js file. I am still working on figuring out what needs to be changed to update the references to functions used in the HTML.

### CODE REVIEW

File: web/js/playground.js [hunk 1]
```diff
@@ -200,4 +199,46 @@ document.addEventListener('DOMContentLoaded', () => {
     } catch (err) {
         console.error("Error initializing the playground: ", err);
     }
-});    
\ No newline at end of file
+});    
+
+/**
```
Comment (haydenkarl22) [inline web/js/playground.js:204]: pretty minor thing but if you could change this comment styling to // id greatly appreciate it.

File: web/tools.html [hunk 2]
```diff
@@ -105,134 +105,15 @@ <h4>Tools</h4>
...
-                            <p>${manifestLink}</p>
-                        </a>
-                    `;
-                    manifestContainer.innerHTML += manifestHTML;
-                });
-            }
-
-        </script>
     </body>
 </html>
```
Comment (haydenkarl22) [inline web/tools.html:119]: So far so good, the comments are very informative as to what your code is doing. Great work!

**Review by alar12 — CHANGES_REQUESTED** [review]
The JavaScript needs to be separated from the HTML and placed into a dedicated JS file. This separation has caused malfunctions, specifically with the menu bar and the tools page. Please ensure that all functionalities work as intended after separating, and verify that the codebase operates seamlessly, maintaining the same functionality as before.

**Review by alar12 — APPROVED** [review]
Internal JS is moved to the external files as intended.

Need to separate tools.html JS from playground.js further!!!

**Pull request merged**